docrawl/docrawl_core.py

```diff
@@ -1022,39 +986,39 @@ def parse(self, response):
 
                     spider_request['loaded'] = True
                     browser_meta_data['request'] = spider_request
-
-                    
-
                     self.docrawl_client.set_browser_meta_data(browser_meta_data)
-
-                if not spider_function['done']:
+                elif not spider_function['done']:
```

Comment:
why joining this if with the previous if? Making it more convoluted

Reply:
Each iteration of the main loop can perform only one action